Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -45,12 +33,12 @@
 # pre-generation of WF numbers
 numWFStart={
     2017: 10000,
-    2023: 20000,
+    2023: 20400,#GB: since the WF numbers are appended from this and I want to skip this one I increased this number to the new actual one 
```

Comment:
I would prefer to keep this as the nominal 20000 and just use the `numWFConflict` to skip it